movie-browser/src/App.js

```diff
@@ -1,12 +1,15 @@
 import React, { Component } from 'react';
 import MovieList from './components/MovieList';
+import Button from './components/Button';
 import { fetchData } from './api';
 
 const URL = 'https://ghibliapi.herokuapp.com/films';
 
 class App extends Component {
   state = {
     movies: [],
+    toggleMovies: 'false',
```

Comment:
this should be a boolean and not a string. It's an on and off scenario, so boolean is ideal. Also, the variable name could be `shouldShowMovies` or `showMovies`, you need to be as explicit as possible.

Reply:
made the changes

Reply:
Something didn't seem quite right with the logic for `{!toggleMovies && <MovieList allMovies={movies} />}`, now I see it why

Reply:
:)